ui/tests/integration/components/mount-backend-form-test.js

```diff
@@ -82,6 +82,18 @@ module('Integration | Component | mount backend form', function (hooks) {
       assert.strictEqual(component.pathValue, 'newpath', 'keeps custom path value');
     });
 
+    test('it does not show a selected token type when first mounting an auth method', async function (assert) {
```

Comment:
Is it maybe worth asserting that the 4 expected token type values also exist in the dropdown?

Reply:
Good idea, I'll amend.